lib/compiler/doc/notes.md

```diff
@@ -2985,7 +2985,7 @@ This document describes the changes made to the Compiler application.
 
   Own Id: OTP-10706
 
-- Binary syntax matches could cause an internal consistency error in in the
+- Binary syntax matches could cause an internal consistency error in the
```

Comment:
Release notes have already been published, there's no use correcting errors here.

Reply:
Sorry for all the inconvenience for such a dumb PR, I'm not used to rebasing and for the fix I text replaced all and I don't know the repository that well :sweat: 

Thanks for the patience :)

Reply:
It's not a dumb PR, see this as an exercise in learning `git rebase` :smile: